lib/LaTeXML/Package/pgfsys-latexml.def.ltxml

```diff
@@ -93,12 +97,14 @@ DefConstructor('\lxSVG@insertpicture{}',
     else {
       $document->openElement('ltx:picture');
       $document->openElement('svg:svg',
-        version  => "1.1",
-        width    => $props{pxwidth}, height => $props{pxheight},
-        viewBox  => "$props{minx} $props{miny} $props{pxwidth} $props{pxheight}",
+        version => "1.1",
+        width   => $props{pxwidth}, height => $props{pxheight},
+####        viewBox  => "$props{minx} $props{miny} $props{pxwidth} $props{pxheight}",
         overflow => "visible");
+      my $x0 = $props{minx}; $x0 = '-' . $x0 unless $x0 =~ s/^-//;
+      my $y0 = $props{miny}; $y0 = '-' . $y0 unless $y0 =~ s/^-//;
```

Comment:
You could also consider the more numeric approach:
```perl
my $x0 = 0+$props{minx};
$x0 = -$x0 if $x0 > 0;
```

Added benefit of avoiding the `-0` edge case.

Reply:
Ah, yes, actually `$x0 = -(0+$props{minx});`

Reply:
wait! but that's different than the string code in the PR? 
```perl
$x0 = '-' . $x0 unless $x0 =~ s/^-//;
```

This will negate a positive `$x0` but will keep a negative `$x0` exactly as it was. Meanwhile your one-liner will negate any number. One of the two was intended, I'm not sure which one :>

Reply:
Ah, the regex had an `s` flag !! I totally missed that. Yes, your one-liner is great indeed.